@xen-orchestra/lite/src/components/ui/UiBadge.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <span class="ui-badge">
-    <FontAwesomeIcon :icon="icon" />
+    <FontAwesomeIcon v-if="icon" :icon="icon" />
```

Comment:
I think we should now use the `UiIcon` component (which also does this check)

```suggestion
    <UiIcon :icon="icon" />
```